__tests__/parse-templates.test.ts

```diff
@@ -578,6 +579,90 @@ describe('parseTemplates', function () {
     expect(templates).toEqual(expected);
   });
 
+  it('with multiple identifiers for the same import path', function () {
```

Comment:
Let's update the title to be something like `with DEFAULT_PARSE_OPTIONS`

Reply:
good call, updated.